tools/tertiary-analysis/cell-types-analysis/.shed.yml

```diff
@@ -0,0 +1,20 @@
+categories:
+ - Transcriptomics
+ - RNA
+ - Statistics
+ - Benchmarking
+description: "Tools for analysis of predicitons from scRNAseq cell type classification tools, based on https://github.com/ebi-gene-expression-group/cell-types-analysis"
```

Comment:
Typo 'predicitons'

Reply:
And it's more 'see https...' than 'based on https...' (i.e. its just providing access to exactly those scripts)